spirv_common.hpp

```diff
@@ -1237,6 +1237,7 @@ struct Meta
 	Decoration decoration;
 	std::vector<Decoration> members;
 	uint32_t sampler = 0;
+	uint32_t image = 0;
```

Comment:
I think I'll need to refactor this to SPIRCombinedSampler that HLSL uses eventually ...